en/core-api/bank-transfer.md

```diff
@@ -1,69 +1,69 @@
 # Core API Bank Transfer Integration
 <hr>
-Basic integration process of Bank Transfer (Virtual Account) will be explained below.
+Basic integration process for Bank Transfer (Virtual Account) is explained in this section.
+Your customers can make payments using the <i>Bank Transfer</i> payment method provided by Midtrans. You will be notified when customer completes the transaction using this option. A list of banks supported by Midtrans is given below. 
+<ol><li>BCA Virtual Account</li>
+    <li>BNI Virtual Account</li>
+    <li>BRI Virtual Account</li>
+    <li>Mandiri Bill Payment</li>
+    <li>Permata Virtual Account</li></ol>
+Although, Midtrans can accept transfer from any other banks, as long as it is real time transfer. 
+For each respective bank, Midtrans creates a virtual account which is allocated to merchant.
```

Comment:
This paragraph is using html tag formating e.g: `<i>, <ol>, <li>` Please avoid this. Use markdown formatting for consistency with other pages.

Put a blank line between the `<hr>` with this paragraph, then you can use markdown formatting.